W/Wayland/build_tarballs.jl

```diff
@@ -48,7 +54,6 @@ dependencies = [
     Dependency("Libffi_jll"; compat="~3.2.2"),
```

Comment:
Can you update this to `compat="~3.4.7"` to make this package work with other recently build packages?

Reply:
Done!